Content.Server/Chemistry/EntitySystems/SolutionContainerSystem.cs

```diff
@@ -103,6 +103,10 @@ private void InitSolution(EntityUid uid, SolutionContainerManagerComponent compo
 
         _appearance.SetData(uid, SolutionContainerVisuals.FillFraction, solution.FillFraction, appearanceComponent);
         _appearance.SetData(uid, SolutionContainerVisuals.Color, solution.GetColor(_prototypeManager), appearanceComponent);
+        if (solution.Name != null)
+        {
+            _appearance.SetData(uid, SolutionContainerVisuals.SolutionName, solution.Name, appearanceComponent);
+        }
```

Comment:
As per abraham's comment this is just going to update the visuals to whatever the latest solution update happens to be.

This should just be something set on solutioncontainervisuals and not networked via appearancedata.